src/tests/suite/SConscript

```diff
@@ -1,6 +1,6 @@
 """Build test suite"""
 import subprocess
-""" hack to handle old subprocess version """
+#hack to handle old subprocess version
```

Comment:
again why a '#' in this case ?

Reply:
pylint doesn't like the """ comments inilne